### PR TITLE
Add an OEP-2 compliant openedx.yaml file

### PR DESCRIPTION
**Pull request by cpennington**
This adds an `openedx.yaml` file, as described by OEP-2:
http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html

The data in this file was transformed from the contents of
edx/repo-tools-data:repos.yaml

### CODE REVIEW

**Comment by cpennington** [comment]
@edx/analytics: I'm marking you folks as the owner of this repo.

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: dash
+oeps: {}
+openedx-release: {ref: release, requirements: requirements/base.txt}
```
Comment (dsjen) [inline openedx.yaml:6]: What does `requirements: requirements/base.txt` do?  I didn't see this on the [oep-2](https://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html) docs and the other oep related PRs don't seem to have this.

Comment (cpennington) [reply openedx.yaml:6]: Check out https://github.com/edx/open-edx-proposals/pull/18/files#diff-13c9b7b8133d4bcf6ab907c7f0065187R91

**Comment by dsjen** [comment]
Thanks!  👍

**Pull request merged**